internal/configs/action.go

```diff
@@ -16,7 +16,7 @@ import (
 type Action struct {
 	Name    string
 	Type    string
-	Config  hcl.Body
+	Config  *hcl.Body
```

Comment:
`hcl.Body` is an interface. Needing a pointer here is probably an indication that something else is going wrong.

Reply:
Ah, I was trying to express it being optional but didn't register that `hcl.Body` is an interface. Will fix it 👍